runtime/doc/eval.txt

```diff
@@ -1386,6 +1386,19 @@ v:errors	Errors found by assert functions, such as |assert_true()|.
 <		If v:errors is set to anything but a list it is made an empty
 		list by the assert function.
 
+					*v:event*
+v:event		Data related to an |autocommand| event as a Dictionary. The
+		available keys differ between different events and are
+		specificed at the documentation for each |event|. The possible
+		keys are:
+	regname 	requested register ["x] or the empty string for an
+			unnamed operation.
+	regcontents	text stored in the register as a list of lines.
```

Comment:
Usually when such list of lines is referenced `readfile()` or `getline()` is also mentioned. E.g. `|readfile()|-style list of lines`.